volume/drivers/fake/fake.go

```diff
@@ -0,0 +1,219 @@
+/*
+Package fake provides an in-memory fake driver implementation
+Copyright 2018 Portworx
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package fake
+
+import (
+	"fmt"
+
+	"github.com/sirupsen/logrus"
+
+	"strings"
+
+	"github.com/libopenstorage/openstorage/api"
+	"github.com/libopenstorage/openstorage/volume"
+	"github.com/libopenstorage/openstorage/volume/drivers/common"
+	"github.com/pborman/uuid"
+	"github.com/portworx/kvdb"
```

Comment:
kvdb code implemented in osd?

Reply:
Not sure what you mean. It is just imported.